profiles/josevale99.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/39961893/213903888-4347babe-71b8-41b3-b69e-836c1fe87503.png",
+    "issueId": 1672,
+    "name": "José Valentín",
+    "username": "JoseVale99"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```